common/frame.go

```diff
@@ -567,7 +567,7 @@ func (f *Frame) waitForFunction(
 
 	execCtx := f.executionContexts[world]
 	if execCtx == nil {
-		return nil, fmt.Errorf("execution context %q not found", world)
+		return nil, fmt.Errorf("execution context %q not found; most likely because of a navigation", world)
```

Comment:
I don't think this is right. A navigation wouldn't cause us to delete a `"main"` or `"utility"` context. They're replaced with contexts with different IDs, not outright deleted.

Reply:
So are you suggesting [this](https://github.com/grafana/xk6-browser/pull/438#discussion_r916713586)?

Reply:
No, that's a different suggestion, where "most likely because of a navigation" probably makes sense. In this case, it doesn't.